samples/Samples.Wcf/Program.cs

```diff
@@ -10,6 +10,11 @@ public static class Program
         private const string WcfPort = "8585";
         private const string WcfNamespace = "WcfSample";
 
+#if NETFRAMEWORK
+        // On .NET Framework, tell the runtime to load assemblies from the GAC domain-neutral.
+        // In this sample, this will affect System.ServiceModel
+        [LoaderOptimization(LoaderOptimization.MultiDomainHost)]
```

Comment:
Do we want this `[LoaderOptimization]` to always apply, or should we test both with and without? cc @zacharycmontoya

Reply:
This PR doesn't address WCF.